specification/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/preview/2023-07-01-preview/redhatopenshift.json

```diff
@@ -0,0 +1,2623 @@
+          "type": "string"
+        },
+        "message": {
+          "description": "A message describing the error, intended to be suitable for display in a user interface.",
+          "type": "string"
+        },
+        "target": {
+          "description": "The target of the particular error. For example, the name of the property in error.",
+          "type": "string"
+        },
+        "details": {
+          "description": "A list of additional details about the error.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudErrorBody"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "ClusterProfile": {
+      "description": "ClusterProfile represents a cluster profile.",
+      "type": "object",
+      "properties": {
+        "pullSecret": {
+          "description": "The pull secret for the cluster.",
+          "type": "string"
+        },
+        "domain": {
+          "description": "The domain for the cluster.",
+          "type": "string"
+        },
+        "version": {
+          "description": "The version of the cluster.",
+          "type": "string"
+        },
+        "resourceGroupId": {
+          "description": "The ID of the cluster resource group.",
+          "type": "string"
+        },
+        "fipsValidatedModules": {
+          "$ref": "#/definitions/FipsValidatedModules",
+          "description": "If FIPS validated crypto modules are used"
+        }
+      }
+    },
+    "ConsoleProfile": {
+      "description": "ConsoleProfile represents a console profile.",
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "The URL to access the cluster console.",
+          "type": "string"
+        }
+      }
+    },
+    "Display": {
+      "description": "Display represents the display details of an operation.",
+      "type": "object",
+      "properties": {
+        "provider": {
+          "description": "Friendly name of the resource provider.",
+          "type": "string"
+        },
+        "resource": {
+          "description": "Resource type on which the operation is performed.",
+          "type": "string"
+        },
+        "operation": {
+          "description": "Operation type: read, write, delete, listKeys/action, etc.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Friendly name of the operation.",
+          "type": "string"
+        }
+      }
+    },
+    "EffectiveOutboundIP": {
+      "description": "EffectiveOutboundIP represents an effective outbound IP resource of the cluster public load balancer.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The fully qualified Azure resource id of an IP address resource.",
+          "type": "string",
+          "format": "arm-id"
```

Comment:
Consider adding "x-ms-arm-id-details" too

https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details

Reply:
it is not clear what is required under  `"x-ms-arm-id-details"` and we have not previously provided this in the code. Given that this PR is time sensitive I believe this should suffice.